sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -614,6 +642,188 @@ export class OfferingValidationModel {
       "Not able to find a program related to this offering or it was not provided.",
   })
   programContext: EducationProgramForOfferingValidationContext;
+
+  /**
+   * Institution information required to execute the offering validation.
+   */
+  @IsOptional()
+  // When the context is provided, it must be valid.
+  @IsNotEmptyObject(undefined, {
+    message: "Not able to find the institution of the offering.",
+  })
+  institutionContext?: InstitutionValidationContext;
```

Comment:
Can this be required following the same validation from the `programContext`?

Reply:
In that case, we have to populate this context for bulk offering upload. Hence I kept it optional.

Reply:
I see, It seems something that it can be avoided now, which is ok for this ticket.